05/app.js

```diff
@@ -1,53 +1,70 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    '/dolor.html': 0,
+  },
 };
 
 /* tutaj umieść swój kod */
 
+const pList = document.querySelectorAll('.text');
+
+const handleClick = function (e) {
+  e.preventDefault();
+  const href = e.target.getAttribute('href');
+  const id = e.target.dataset.id;
+  console.log(href);
+
+  if (href in stats.links) {
```

Comment:
Tutaj jest problem. Przeczytaj sobie co ten warunek oznacza: "Jeśli istnieje href na liście statystyk to zaktualizuj wartość", w tym przypadku warunek ma być inny. "Jeśli nie istnieje href na liście statystyk to go utwórz", potem dodaj jeden do obecnej wartości czyli:
```
if (typeof stats.links[href] === 'undefined') { // mozna też zrobic z operatorem in
    stats.links[href] = 0;
}

stats.links[href]++;
```
Podobnie trzeba zrobić z paragrafem :)